src/test/java/com/google/api/codegen/testsrc/libraryproto/library_gapic.yaml

```diff
@@ -262,6 +262,7 @@ interfaces:
       - comment:
         - "Do something with %s one by one"
         - book_names
+      - define: book_names = $resp.book_names
```

Comment:
Suggest you add a `- comment` saying you're just defining this variable for baseline testing purposes, even though it's not subsequently used.